packages/tabs/src/Tabs.spec.tsx

```diff
@@ -122,67 +129,90 @@ describe('packages/tabs', () => {
       expect(screen.queryByText('Content 1')).not.toBeInTheDocument();
     });
 
-    test('selected tab is active on first render', () => {
-      renderTabs({ setSelected, selected: 1 });
-      const activeTab = screen.getByText('Content 2');
-      expect(activeTab).toBeVisible();
+    test('selected tab panel is active on first render', () => {
+      const { getSelectedPanel } = renderTabs({ setSelected, selected: 1 });
+      const selectedPanel = getSelectedPanel();
+      expect(selectedPanel).toHaveTextContent('Content 2');
     });
 
-    test('clicking a tab does not change the active tab', () => {
-      renderTabs({ setSelected, selected: 1 });
-      const tab = screen.getByText('First');
-      fireEvent.click(tab);
+    test('clicking a tab does not change the selected tab panel', () => {
```

Comment:
Why doesn't clicking on a tab change the tab content?

Reply:
I didn't write the original test, but I think it's because it's a controlled input so it's dependent on whatever `setSelected` does. In this test block, `setSelected` is just a jest mock, so it doesn't actually update `selected` index

Reply:
got it